src/main/java/com/petrych/screenshotter/persistence/model/Screenshot.java

```diff
@@ -86,13 +99,14 @@ public boolean equals(Object o) {
 		return id.equals(that.id) &&
 				name.equals(that.name) &&
 				uri.equals(that.uri) &&
-				dateTimeCreated.equals(that.dateTimeCreated);
+				dateTimeCreated.equals(that.dateTimeCreated) &&
```

Comment:
Have you looked into Lombok? :D https://projectlombok.org/ 

It's really nice for this boiler plate stuff and ensures that methods are up to date when adding new fields to your classes. Example, the `toString` in this file doesn't include `fileName`. Maybe that's the intention though? 🤔

Reply:
Yes, in the toString() it's the intention. Because it exposes info to a client, and I want to keep fileName as implementation details